docs/resources/kargo_instance.md

```diff
@@ -114,6 +114,18 @@ EOT
       }
     }
   }
+  kargo_resources = local.kargo_resources
+}
+
+locals {
+  yaml_files = fileset("${path.module}/kargo-manifests", "*.yaml")
+
+  kargo_resources = flatten([
+    for file_name in local.yaml_files : [
+      for resource in split("\n---\n", file("${path.module}/kargo-manifests/${file_name}")) :
+      jsonencode(yamldecode(resource))
+    ]
+  ])
 }
 ```
```

Comment:
I think it will be useful if we could include some of the content of `examples/resources/akp_kargo_instance/kargo-manifests/kargo.yaml` to the docs as well so that user can know how the yaml manfiests look like since I think the `kargo.yaml` won't be included in https://registry.terraform.io/providers/akuity/akp/latest/docs by default?

Reply:
Added, thank you!